tests/test_examples.py

```diff
@@ -47,6 +49,16 @@ def test_examples(example_dir):
     actual_dir = example_dir.joinpath("output", "actual")
     dep_file_path = example_dir.joinpath("dependencies.yaml")
 
+    # Copy pyproject.toml files from expected to actual since they are modified in place
+    for dirpath, _, filenames in os.walk(expected_dir):
+        for filename in filenames:
+            if filename == "pyproject.toml":
+                full_path = pathlib.PurePath(dirpath) / filename
```

Comment:
Why are you using `PurePath`? We expect the paths to exist, no?

Reply:
Yup will change.